### DISCUSSION TITLE
Model.update has no `replacements`?

### DISCUSSION THREAD

**Discussion by 1valdis** [sequelize/sequelize#17266] [Help & Questions]
Consider the following code:
```ts
    ChangeModel.update(
      {
        isRead: true,
      },
      {
        where: {
          id: idInTeam,
          teamId: {
            [Op.in]: Sequelize.literal(
              `(SELECT team_id FROM teams WHERE responsible_employee_id = :responsibleEmployeeId)`,
            ),
          },
        },
      },
    );
```
This will not work because unfortunately here I cannot pass `replacements` along with `where` in the `UpdateOptions`, unlike if that would be a `findAll`. So instead I have to sanitize responsibleEmployeeId myself and pass it into the template string? Is there a better way? Is it worth doing feature request?

**Comment by AKikhaev** [comment]
After bunch of different ways to do that with errors like:
```
"error": "Both `replacements` and `bind` cannot be set at the same time"
"error": "Both `sql.bind` and `options.bind` cannot be set at the same time"
```
I can suggest a solution:
```
[Op.in]: Sequelize.literal(
              `(SELECT team_id FROM teams WHERE responsible_employee_id = ${sequelize.escape(responsibleEmployeeId)})`,
            ),
```

sequelize is an instance of your `new Sequelize(...)`
